### ISSUE TITLE
PCF Tree Component configuration

### ISSUE THREAD

**Issue by walcivar** [opc-cpvp/OPC.PowerApps.PCFControls#863]
Hi @joshcampbell191, @GabrielFG, @StephBrouil, @pierluc-ssoucy, @JonathanThomassin, @OPCSheldon

I don't know how to configure the Tree Component PCF in the form. The documentation is not the best to understand the configuration, could you help me to understand at least what I have to put in the required parameters please?

Let me give a little bit of my context:

Principal entity: Lead (lead)
Related Entity: Customer Classification (dp_customer_classification). This is a custom entity.
Relationship between these two entities: Many to many (dp_lead_dp_customer_classification)

The Customer Classification entity has a lookup field that aims to the same entity to set up the parent-child relationship. (dp_parent_classification)

The PCF currently has this configuration:
Tree Entity Collection Name: dp_customer_classifications (plural name of the custom entity)
Tree parent attribute: dp_parent_classification (name of the lookup field in the custom entity)
Tree Name attribute: dp_name (name field of the custom entity)
Tree Id attribute: dp_customer_classificationid (Guid field of the custom entity)
Relationship Name: dp_lead_dp_customer_classification (name of the relationship)
Relationship Entity: dp_lead_dp_customer_classification (its the same name of the relationship)

![PCFConfig](https://user-images.githubusercontent.com/5630463/190135246-fb262520-a6f1-412f-b798-45f78983ae9e.gif)

This is what I get after configuring the PCF:

![PCFResult](https://user-images.githubusercontent.com/5630463/190136610-cd6413dc-4487-40c0-9208-97b33691d8c8.gif)

Can you help me understand what I am doing wrong?

**Comment by Niels-SL** [comment]
@walcivar did you get the tree component to work? It looks cool, but I am not able to get it to work. 

Tagging a few others who have created similar issues, @JeroenHensmans, @simplesi

**Comment by walcivar** [comment]
@Niels-SL 
No, I gave up trying. But here is another PCF that displays JSON information as a tree view. The difference of this one is that it doesn't have the select node or the search function.

https://github.com/walcivar/JsonTreeViewer

**Comment by Niels-SL** [comment]
Too bad. Thanks for the link to your [JsonTreeViewer](https://github.com/walcivar/JsonTreeViewer). 

There are also some other options on [pcf.gallery](https://pcf.gallery/) such as the [Tree Relatsionships](https://pcf.gallery/tree-relationships/). However, I really liked the search functionality of this one.

**Comment by Rednas1988** [comment]
I managed to get it working by using the EntitySetName instead of the PluralName of the CustomEntity. I also configured it via the old interface. By default PluralName and EntitySetName are the same, but somehow they got mixed up.
